tutorials/tour/abstract-types.md

```diff
@@ -5,11 +5,10 @@ title: Abstract Types
 disqus: true
 
 tutorial: scala-tour
-num: 2
-outof: 35
```

Comment:
Haha funny that we have 33 not 35. No comment there (I must be responsible). `outof` is used for generating indicies and TOCs... Liquid markup is not the smartest templating engine – last I checked you couldn't populate a data structure in it with a subset of pages you want, so we have to do a very dumb "x out of y" articles kind of thing to know when to stop looping through all pages in the site. Yeah. I know.